src/Domain/DomainHelpers.hpp

```diff
@@ -42,13 +42,12 @@ template <typename T>
 struct create_from_yaml;
 }  // namespace Options
 namespace domain::CoordinateMaps {
-template<typename Map1, typename Map2>
+template <typename Map1, typename Map2>
 class ProductOf2Maps;
-template<typename Map1, typename Map2, typename Map3>
+template <typename Map1, typename Map2, typename Map3>
 class ProductOf3Maps;
-class Affine;
-class Equiangular;
-template<size_t Dim>
+class Interval;
+template <size_t Dim>
 class Wedge;
```

Comment:
I'm going through the commits again, checking the diffs. Is the deletion of Affine and Equiangular correct by the way?